spec.html

```diff
@@ -35921,17 +36540,21 @@ <h1>
             Static Semantics: ParsePattern (
               _patternText_: a sequence of Unicode code points,
               _u_: a Boolean,
+              _v_: a Boolean,
             )
           </h1>
           <dl class="header">
           </dl>
           <emu-alg>
-            1. If _u_ is *true*, then
-              1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+UnicodeMode, +N]|).
+            1. If _v_ is *true* and _u_ is *true*, throw a *SyntaxError* exception.
```

Comment:
`ParsePattern` (like `ParseText`) signals a parse error by returning a List of SyntaxError objects rather than by throwing an exception.
```suggestion
            1. If _v_ is *true* and _u_ is *true*, then
              1. Let _parseResult_ be a List containing one *SyntaxError* object.
            1. Else if ...
```